programs/mango-v4/src/state/group.rs

```diff
@@ -4,8 +4,25 @@ use std::mem::size_of;
 
 // TODO: Assuming we allow up to 65536 different tokens
 pub type TokenIndex = u16;
+
+/// This token index is supposed to be the token that oracles quote in.
+///
+/// In practice this is set to the USDC token index, and that is wrong: actually
```

Comment:
so this comment will go away, when we do the big change, right? when the quoting/tick comes in usdc

Reply:
The whole QUOTE_TOKEN_INDEX constant needs to go away. There's no registered token that matches the oracle quote currency.